tests/Foundation/FoundationHelpersTest.php

```diff
@@ -83,4 +83,29 @@ public function testMixDoesNotIncludeHost()
 
         unlink(public_path('mix-manifest.json'));
     }
+
+    public function testMixReturnsHotUrl()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__;
+        });
+
+        $hotFile = public_path('hot');
+
+        try {
+            file_put_contents($hotFile, 'http://localhost:8080/');
+
+            $this->assertEquals('//localhost:8080/app.css', mix('app.css'));
```

Comment:
Any reason not to use `assertSame()`?

Reply:
For what? We do not know exactly which type will return the mix() helper. But we know that it can be converted to a string.